app/controllers/vat/VatAccountUnavailableController.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.vat
+
+import config.FrontendAppConfig
+import controllers.actions.{AuthAction, ServiceInfoAction}
+import javax.inject.Inject
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import views.html.vat.vatAccountUnavailable
+
+class VatAccountUnavailableController @Inject()(appConfig: FrontendAppConfig,
+                                                mcc: MessagesControllerComponents,
+                                                authenticate: AuthAction,
+                                                vatAccountUnavailable: vatAccountUnavailable)
+  extends FrontendController(mcc) with I18nSupport {
+
+  def onPageLoad: Action[AnyContent] = authenticate { implicit request =>
+    Ok(vatAccountUnavailable(appConfig))
+  }
+
+}
```

Comment:
This probably didn't need to be a new controller by the way. The onPageLoad function here could have been added to `WhatIsYourVATRegNumberController` to make it easier for you.